src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -52,6 +63,25 @@
            [:div {} k [:span {:style {:margin "0 1em"}} "→"] (display-value v)])])])})
 
 
+(react/defc WorkflowTiming
+  {:get-initial-state
+   (fn []
+     {:expanded false})
+   :render
+   (fn [{:keys [props state]}]
+     [:div {}
+      (create-field
+        (:label props)
+        (if (empty? (:data props))
+          "Not Available"
+          (style/create-link {:text (if (:expanded @state) "Hide" "Show")
+                              :onClick #(swap! state assoc :expanded (not (:expanded @state)))})))
```

Comment:
Can simplify this onClick to `#(swap! state update :expanded not)`.  Also, it would be more idiomatic to name it `:expanded?`.